src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck started its work");

```

Comment:
The message in the `doWork` method is informative, but make sure it clearly indicates the type of `Machine` as required by the checklist.